LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2004-2016 Castle Project - http://www.castleproject.org/
```

Comment:
As we did in Windsor to work around NuGet not supporting files without an extension, we renamed LICENSE to LICENSE.txt rather than just copy it. Still not sure how we've not noticed this before.